packages/js/product-editor/src/components/iframe-editor/iframe-editor.tsx

```diff
@@ -159,7 +173,7 @@ export function IframeEditor( {
 						setTemporalBlocks( updatedBlocks );
 						onInput( updatedBlocks );
 					} }
-					useSubRegistry={ true }
+					useSubRegistry={ false }
```

Comment:
I don't understand why changing this causes the ComplementaryArea toggling to work.

Reply:
By default, each editor instance creates its own nested store. This means that the modal editor will have its own one and won't share data with the Product editor, and vice versa.

I think there are two options available to us. The first is to use the global store by setting `useSubRegistry={ false }`. This will allow both editor instances to share data, making things easier in the short term.

Alternatively, we can create a new store to share data between both instances.

[The core team suggested](https://github.com/woocommerce/woocommerce/discussions/42761#discussioncomment-7867271) that we avoid using the global store.

Reply:
The big thing I don't understand is why `useSubRegistry` has an impact on how `ComplementaryArea` toggling works (or doesn't).